rsync: change_dir "/home/ue4/carla//./Plugins" failed: No such file or directory (2)

I'm trying to build the docker image, and I am at the last step. 
`docker build -t carla -f Carla.Dockerfile .`
But it gives the error below. 
I am on Ubuntu 18.04, and the carla version is whichever is used by the dockerfile. 

I included some lines above for context, not sure if that helps. 
```
Sumo/util/create_sumo_vtypes.py
Sumo/util/netconvert_carla.py
Sumo/util/sequential_types.py
Sumo/util/data/
Sumo/util/data/opendrive_netconvert.typ.xml
rsync: change_dir "/home/ue4/carla//./Plugins" failed: No such file or directory (2)
rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1196) [sender=3.1.2]
make: *** [package] Error 23
Util/BuildTools/Linux.mk:16: recipe for target 'package' failed
The command '/bin/sh -c cd /home/ue4 &&   if [ -z ${GIT_BRANCH+x} ]; then git clone --depth 1 https://github.com/carla-
simulator/carla.git;   else git clone --depth 1 --branch $GIT_BRANCH https://github.com/carla-simulator/carla.git; fi &&   
cd /home/ue4/carla &&   ./Update.sh &&   make CarlaUE4Editor &&   make PythonAPI &&   make build.utils &&   
make package &&   rm -r /home/ue4/carla/Dist' returned a non-zero code: 2
```

--- a/Util/BuildTools/Package.bat
+++ b/Util/BuildTools/Package.bat
@@ -173,9 +173,8 @@ if %DO_COPY_FILES%==true (
     echo f | xcopy /y "!XCOPY_FROM!Docs\release_readme.md"                          "!XCOPY_TO!README"
     echo f | xcopy /y "!XCOPY_FROM!Util\Docker\Release.Dockerfile"                  "!XCOPY_TO!Dockerfile"
     echo f | xcopy /y "!XCOPY_FROM!PythonAPI\carla\dist\*.egg"                      "!XCOPY_TO!PythonAPI\carla\dist\"
-    echo f | xcopy /y /s "!XCOPY_FROM!PythonAPI\carla\data\*"                          "!XCOPY_TO!PythonAPI\carla\data\"
+    echo f | xcopy /y /s "!XCOPY_FROM!PythonAPI\carla\data\*"                       "!XCOPY_TO!PythonAPI\carla\data\"
     echo d | xcopy /y /s "!XCOPY_FROM!Co-Simulation"                                "!XCOPY_TO!Co-Simulation"
-    echo d | xcopy /y /s "!XCOPY_FROM!Plugins"                                      "!XCOPY_TO!Plugins"
     echo d | xcopy /y /s "!XCOPY_FROM!PythonAPI\carla\agents"                       "!XCOPY_TO!PythonAPI\carla\agents"
     echo f | xcopy /y "!XCOPY_FROM!PythonAPI\carla\scene_layout.py"                 "!XCOPY_TO!PythonAPI\carla\"
     echo f | xcopy /y "!XCOPY_FROM!PythonAPI\carla\requirements.txt"                "!XCOPY_TO!PythonAPI\carla\"
@@ -186,6 +185,9 @@ if %DO_COPY_FILES%==true (
     echo f | xcopy /y "!XCOPY_FROM!PythonAPI\util\requirements.txt"                 "!XCOPY_TO!PythonAPI\util\"
     echo f | xcopy /y "!XCOPY_FROM!Unreal\CarlaUE4\Content\Carla\HDMaps\*.pcd"      "!XCOPY_TO!HDMaps\"
     echo f | xcopy /y "!XCOPY_FROM!Unreal\CarlaUE4\Content\Carla\HDMaps\Readme.md"  "!XCOPY_TO!HDMaps\README"
+    if exist "!XCOPY_FROM!Plugins" (
+        echo d | xcopy /y /s "!XCOPY_FROM!Plugins"                                  "!XCOPY_TO!Plugins"
+    )
 )
 
 rem ==============================================================================

--- a/Util/BuildTools/Package.sh
+++ b/Util/BuildTools/Package.sh
@@ -163,7 +163,9 @@ if ${DO_CARLA_RELEASE} ; then
 
   copy_if_changed "./Co-Simulation/" "${DESTINATION}/Co-Simulation/"
 
-  copy_if_changed "./Plugins/" "${DESTINATION}/Plugins/"
+  if [ -d "./Plugins/" ] ; then
+    copy_if_changed "./Plugins/" "${DESTINATION}/Plugins/"
+  fi
 
   copy_if_changed "./Unreal/CarlaUE4/Content/Carla/HDMaps/*.pcd" "${DESTINATION}/HDMaps/"
   copy_if_changed "./Unreal/CarlaUE4/Content/Carla/HDMaps/Readme.md" "${DESTINATION}/HDMaps/README"

